pallets/replace/src/types.rs

```diff
@@ -4,7 +4,7 @@ use frame_support::traits::Get;
 pub use primitives::replace::{ReplaceRequest, ReplaceRequestStatus};
 use primitives::VaultId;
 use sp_runtime::DispatchError;
-use vault_registry::types::CurrencyId;
+pub use vault_registry::types::CurrencyId;
```

Comment:
All other pallets expose this type. for example` issue`, `redeem` pallets contain
`pub use vault_registry::types::CurrencyId;`
We can't leave it without `pub`.
because of the error.

```
pallets/replace/src/ext.rs:11:20
   |
11 |     use crate::types::CurrencyId;
   |                       ^^^^^^^^^^ private type alias import
```

So this is just a solution that introduced in other pallets.